src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManager.java

```diff
@@ -673,4 +673,10 @@ protected ExecutorPair(E executor, boolean automaticShutdown)
             this.automaticShutdown = automaticShutdown;
         }
     }
+
+    @Override
+    public boolean isShutdown()
+    {
+        return this.shutdown.get();
```

Comment:
See [initial discussion in Discord](https://discord.com/channels/125227483518861312/125227483518861312/1125903549847584798). This is a logic error.